Documentation/tutorial/validate_your_markdown_files.md

```diff
@@ -1,24 +1,25 @@
-# Validate Your Markdown Files
+# Validate your Markdown files
 
-In Markdown, we can write any document with valid syntax. For example, Markdown supports to directly write HTML tag, we can write HTML tag `<h1>title</h1>` instead of Markdown syntax `#title`.
-But for some purpose, some behaviors are unwanted, for example, you may not want to allow `<script>` tag in Markdown that can insert any javascript.
+In Markdown, it is possible to write any type of content, as long as the used syntax is valid. For example, Markdown supports the direct use of HTML tags - one can use the `<h1>title</h1>` syntax instead of conventional Markdown, such as `#title`.
 
-In this document, you'll learn how to define markdown validation rules, which will help you to validate markdown documents in an efficient way.
+With full-fledged HTML support, some behaviors might not be desirable. For example, you may not want to allow `<script>` tags included in Markdown, as that can introduce arbitrary JavaScript into documentation.
 
-> [!Note]
-> Markdown validation is part of DFM, if you switch Markdown engine to other engine, validation might not work.
+In this document, you'll learn how to define Markdown validation rules, which will help you ensure that your document follows strict conventions.
 
-There're three kinds of validation rules provided by DocFX:
+>[!NOTE]
+>Markdown validation is part of the Markdown processor in DocFX. If you switch the Markdown engine, validation rules might not apply the same way.
```

Comment:
DocFX has several engines, like `dfm`, `markdig`. This note want to let users know that all these steps can only work in `dfm`. We should mention `dfm` here.

Reply:
Fixed in 3e2f8fa